src/tests/testEvent.cc

```diff
@@ -66,20 +60,23 @@ testEvent::testCancel()
     CPPUNIT_ASSERT_EQUAL(0, event_to_cancel.calls);
 }
 
-/* submit two callbacks, and then dump the queue.
- */
+// submit two callbacks, and then dump the queue.
 void
 testEvent::testDump()
 {
     EventScheduler scheduler;
     CalledEvent event;
     CalledEvent event2;
-    CapturingStoreEntry * anEntry = new CapturingStoreEntry();
-    String expect =  "Last event to run: last event\n"
-                     "\n"
-                     "Operation                \tNext Execution \tWeight\tCallback Valid?\n"
-                     "test event               \t0.000 sec\t    0\t N/A\n"
-                     "test event2              \t0.000 sec\t    0\t N/A\n";
+    const char *expected = "Last event to run: last event\n"
+                           "\n"
+                           "Operation                \tNext Execution \tWeight\tCallback Valid?\n"
+                           "test event               \t0.000 sec\t    0\t N/A\n"
+                           "test event2              \t0.000 sec\t    0\t N/A\n";
+    MemBuf expect;
+    expect.init();
+    expect.append(expected, strlen(expected));
+    CPPUNIT_ASSERT_EQUAL(strlen(expected), size_t(expect.contentSize()));
+    CPPUNIT_ASSERT(strcmp(expect.content(), expected) == 0);
```

Comment:
I do not think we should check that MemBuf is working correctly in _this_ unit test. These are basic operations/manipulations that this unit test case can (and, hence, should) take for granted.

```suggestion
```